build/mac/tweak_info_plist.py

```diff
@@ -110,9 +108,6 @@ def Main(argv):
 
   plist['CrProductDirName'] = options.brave_product_dir_name
 
-  if options.brave_feed_url:
```

Comment:
so we're going to lose the option to override? QA team might need a way to test updates on internal channels

Reply:
Ah, I see. I'll revert this change and will use this if provided over generated url.
I'll f/u tomorrow.

Reply:
QA team edits Info.plist to use intenal url?

Reply:
Curious which one is more convenient among editing Info.plist or passing url as command lind args.

Reply:
cc @brave/legacy_qa

Reply:
Command line arg better since it doesn't change the hash or the contents of the app dir and it's more visible.

Reply:
Added `--update-feed-url` switch.

Reply:
@mihaiplesa,  should we get security review for this change?

Reply:
I think we're fine.

Reply:
Ok!